src/openforms/forms/tests/test_api_import_export.py

```diff
@@ -183,7 +186,14 @@ def test_form_import(self):
         self.assertNotEqual(imported_form.pk, form1.pk)
         self.assertNotEqual(imported_form.uuid, str(form1.uuid))
         self.assertEqual(imported_form.active, False)
-        self.assertEqual(imported_form.registration_backend, form1.registration_backend)
+        for imported, original in zip(
+            imported_form.registration_backends.all(),
+            form1.registration_backends.all(),
+        ):
```

Comment:
this is a bit fragile because there's no explicit ordering specified, not in the qs here nor in the model meta. So I'd add an `order_by("id")` here for test robustness.

It's now up to the DB engine to determine in which order results are returned.